samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideLINQ/CS/csRef30LangFeatures_2.cs

```diff
@@ -15,14 +15,14 @@ class Customer
         // Auto-implemented properties for trivial get and set
         public double TotalPurchases { get; set; }
         public string Name { get; set; }
-        public int CustomerID { get; set; }
+        public int CustomerId { get; set; }
 
         // Constructor
         public Customer(double purchases, string name, int ID)
```

Comment:
parameter name

Reply:
I agree with @NextTurn - @Youssef1313 are you able to update the parameter name `ID` as well?